wizard-frontend/tsconfig.json

```
@@ -0,0 +1,19 @@
+{
+  "files": [],
+  "references": [
+    { "path": "./tsconfig.app.json" },
+    { "path": "./tsconfig.node.json" }
+  ],
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./src/*"]
+    },
+    "noImplicitAny": false,
+    "noUnusedParameters": false,
+    "skipLibCheck": true,
+    "allowJs": true,
+    "noUnusedLocals": false,
+    "strictNullChecks": false
+  }
```

```suggestion
    "noImplicitAny": true,
    "noUnusedParameters": true,
    "skipLibCheck": true,
    "allowJs": false,
    "noUnusedLocals": true,
    "strictNullChecks": true
  }
```

Do we need both this and `tsconfig.app.json`?

---

Nice.